tools/data-handler/src/validate.ts

```diff
@@ -158,6 +201,35 @@ export class Validate {
     return message;
   }
 
+  // Removes same items from an array.
+  private removeDuplicateEntries(
+    value: string,
+    index: number,
+    array: string[],
+  ) {
+    return array.indexOf(value) === index;
+  }
+
+  private async validateArrayOfFields(
+    project: Project,
+    cardType: CardType | undefined,
+    fieldArray: string[] | undefined,
```

Comment:
I think it would be generally better to handle undefined values at higher levels rather than at lower levels

Reply:
Fixed. Caller now handles `undefined` and uses empty string arrays if values are missing.